Jenkinsfile

```diff
@@ -16,7 +16,7 @@
 
 def podfile(podfileDir) {
     // Lock the pod repo and update the pod
```

Comment:
It would be good to update this comment with something that focuses on "lock the pod repo per host instead of globally to avoid contention".